digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -48,11 +49,17 @@ object ProjectState {
   case object Deleted extends ProjectState {def value = 7; def description = "Poistettu projekti"}
 
   case object ErrorInViite extends ProjectState {
-    def value = 8
+    def value = 8;
```

Comment:
No need for semicolons in Scala.

Reply:
God dammit... I blame switching too often :)